Tasks/CloudFormationCreateOrUpdateStack/task.json

```
@@ -224,6 +224,15 @@
             "groupName": "Capabilities",
             "helpMarkDown": "Capabilities that must be specified before AWS CloudFormation can update certain stacks. Some stack templates might include resources that can affect permissions in your AWS account, for example, by creating new AWS Identity and Access Management (IAM) users. For those stacks, you must explicitly acknowledge their capabilities by specifying this parameter.\n\nIf your stack manipulates IAM resources with custom names, you must add this capability otherwise an InsufficientCapabilities error will be returned."
         },
+        {
+            "name": "capabilityAutoExpand",
+            "type": "boolean",
+            "label": "Allow use of CloudFormation Macros ('CAPABILITY_AUTO_EXPAND')",
+            "defaultValue": "true",
```

Thanks for the contribution!

The only thing I am worried about is this is set to true by default, I'll have to check if this is an issue or not.

---

Would changing it's default to false allow this to be accepted?
```suggestion
            "defaultValue": "false",
```